charts/osm/templates/mutatingwebhook.yaml

```diff
@@ -3,17 +3,28 @@ kind: MutatingWebhookConfiguration
 metadata:
   labels:
     app: osm-controller
-  name: osm-webhook-{{.Values.OpenServiceMesh.meshName}}
+  name: arc-osm-webhook-{{.Values.OpenServiceMesh.meshName}}
 webhooks:
 - name: osm-inject.k8s.io
   clientConfig:
     service:
       name: osm-controller
-      namespace: {{.Release.Namespace}}
+      namespace: arc-osm-system
```

Comment:
can we get this value from values.yaml instead? then default to `arc-osm-system` in values.yaml

Reply:
I have added a new field to values.yaml and set the default to `arc-osm-system`